model/registry/system.yaml

```diff
@@ -0,0 +1,255 @@
+groups:
+  # General system attributes
+  - id: registry.system
+    prefix: system
+    type: attribute_group
+    brief: "Describes System metric attributes"
+    attributes:
+      - id: device
+        type: string
+        stability: experimental
+        brief: "The device identifier"
+        examples: ["(identifier)"]
+  # system.cpu.* attribute group
+  - id: registry.system.cpu
+    prefix: system.cpu
+    type: attribute_group
+    brief: "Describes System CPU metric attributes"
```

Comment:
To https://github.com/open-telemetry/semantic-conventions/pull/867#discussion_r1549724033:

I think the `brief`s here are different compared to the `*.cpu.state` discussions: How about we remove all the `metric`s from the `brief`s for these attributes (because those are indeed problematic) and then we leave the `*.cpu.state` discussion to https://github.com/open-telemetry/semantic-conventions/issues/840?